mon-pix/tests/acceptance/inscription_test.js

```diff
@@ -0,0 +1,72 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, currentURL } from '@ember/test-helpers';
+import { visit } from '@1024pix/ember-testing-library';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import setupIntl from '../helpers/setup-intl';
+
+module('Acceptance | Inscription', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, ['fr', 'en']);
+
+  module('International domain (.org)', function () {
+    module('when accessing the inscription page with "Français" as default language', function () {
+      test('displays the inscription page with "Français" as selected language', async function (assert) {
+        // given & when
+        const screen = await visit('/inscription');
+
+        // then
+        assert.strictEqual(currentURL(), '/inscription');
+        assert.dom(screen.getByRole('heading', { name: this.intl.t('pages.sign-up.first-title'), level: 1 })).exists();
```

Comment:
Remarque : je crois que ce qui était attendu est de trouver le texte en français, là ça passe par this.intl.t et donc on vérifie pas que c'est bien traduit (ça teste un peu le comportement d'intl c'est vrai)